### ISSUE TITLE
rollup-plugin-postcss breaking changes

### ISSUE THREAD

**Issue by dkodeit** [dkodeit/svelte-sapper-template-tailwindcss#1]
Updating version past 2.9.0 breaks the ability to extract outside the output directory.

works for version 2.9.0
https://github.com/dkodeit/svelte-sapper-template-tailwindcss/blob/7808795ba96370eb3fcaa5c0742e79e48417a4ac/rollup.config.js#L110

Version 3+ will not allow the user of relative or absolute paths, wants:
`extract: 'static/tailwind.css'` 

This breaking change results in the tailwind.css file not being extracted to the static directory but instead it's extracted to a directory within __sapper__/(dev, build, export)

In a nutshell no stylesheet in static, broken link within index.html, and no styles during development.

Until this is addressed, will not update past 2.9 and just deal with the warning during build.

https://github.com/rollup/rollup/issues/3669
